src/api-reference/request/request.markdown

```diff
@@ -1,244 +1,411 @@
 ---
-title: Request
+title: Request endpoints
 layout: reference
 ---
 
-# Request
+# Request endpoints
 
-Concur Request automates the spend request and approval process for both travel and everyday expenses, giving you the data you need to accurately track and better control spending. By increasing visibility into planned expenses and up-to-date budget data, you can make strategic spending decisions before any spending actually occurs. The Request resource provides many abilities, including viewing requests and transition of requests into the workflow.
+- [Resources](#resource)
+  - [Expense](#expense-resource)
```

Comment:
_note: switch order with the following if this makes sense also for you:
Request
Workflow
Expense
Travel Agency_